### PR TITLE
Add dump_ops_enable and optrack_toggle_delay options

### PR DESCRIPTION
**Pull request by dzafman**
Add dump_ops_enable which continuously dumps ops using 3 commands
Add optrack_toggle_delay to alternate op tracking enablement

### CODE REVIEW

**Comment by dzafman** [comment]
@liewegas Not sure if this should be on the Jewel branch instead of master.  This tests ceph#8044 which passed my testing using wip-zafman-testing.

**Comment by dzafman** [comment]
While this was working it didn't pass flake8, so I fixed that and just to be sure I'm running a single job to test.

**Comment by dzafman** [comment]
@liewegas @athanatos Ready to merge unless it needs to be rebased moved to Jewel branch.

**Comment by athanatos** [comment]
I think sjust@teuthology:/a/dzafman-2016-03-22_02:15:38-rados:thrash-wip-zafman-testing---basic-smithi/80901 hung waiting to join the new background task.  Presumably it's because the injectargs hung?  I think you'll have to add some kind of timeout to make sure that can't happen.

**Comment by dzafman** [comment]
This must go in before ceph/ceph#7289 or tests will fail.

File: tasks/scrub_test.py [hunk 1]
```diff
@@ -236,9 +236,10 @@ def _get_attrs(self, inc, attr_name):
             log.info('shard = %r' % shard)
             log.info('attr = %s' % attr_name)
             assert 'osd' in shard
-            assert attr_name in shard
             osd = shard['osd']
-            attr = shard[attr_name]
+            attr = shard.get(attr_name)
```
Comment (tchaikov) [inline tasks/scrub_test.py]: could put
```py
attr = shard.get(attr_name, False)
```

**Comment by athanatos** [comment]
lgtm

**Pull request merged**